include/deal.II/lac/block_linear_operator.h

```diff
@@ -347,137 +347,218 @@ class BlockLinearOperator
 
 
 
-namespace internal
+namespace
```

Comment:
No, bad idea -- we don't want to use anonymous namespaces in header files. Can you please undo this change?